### ISSUE TITLE
Dart  not appear

### ISSUE THREAD

**Issue by wav555** [wav555/159333-project#3]
I noticed that the darts in the game are not appearing, is there something wrong with the modifications?

**Comment by 2528334939** [comment]
Yes! I have fixed the problem. There was something wrong with the random number generator.
